### PR TITLE
docs[patch]: Add more distinct deprecation warning in api refs

### PR DESCRIPTION
**Pull request by bracesproul**
image

### CODE REVIEW

File: docs/api_refs/typedoc_plugins/hide_underscore_lc.js [hunk 1]
```diff
@@ -120,15 +130,38 @@ function load(application) {
...
+
+        // There is only 1 <h1> tag in the file, the symbol name. Eg AgentExecutor.
+        // We want to add the `deprecated` class to this tag so it has a strikethrough.
+        htmlFileContent = htmlFileContent.replace(`<h1>`, `<h1 class="deprecated">`);
+      }
+
       const [part1, part2] = htmlFileContent.split(htmlToSplitAt);
       const htmlWithScript = part1 + SCRIPT_HTML + part2;
-      await fsPromises.writeFile(indexFilePath, htmlWithScript);
+      fs.writeFileSync(indexFilePath, htmlWithScript);
```
Comment (jacoblee93) [inline docs/api_refs/typedoc_plugins/hide_underscore_lc.js:164]: Why this change?

File: docs/api_refs/typedoc_plugins/hide_underscore_lc.js [hunk 2]
```diff
@@ -22,6 +21,17 @@ const SCRIPT_HTML = `<script>
...
 </script>`;
 
+/**
+ * @param {string | undefined} deprecationText 
+ * @returns {string}
+ */
+const DEPRECATION_HTML = (deprecationText) => `<div class="deprecation-warning">
+<h2>⚠️ Deprecated ⚠️</h2>
+${deprecationText ? `<p>${deprecationText}</p>` : ""}
+<p>This method is deprecated and will be removed from the LangChain.js library soon.</p>
```
Comment (jacoblee93) [inline docs/api_refs/typedoc_plugins/hide_underscore_lc.js:31]: Let's just change this to:
```
This method is deprecated and will be removed in the future.
```

File: docs/api_refs/package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "postcss": "^8",
     "tailwindcss": "^3.3.0",
     "ts-morph": "^20.0.0",
+    "typedoc": "^0.25.6",
```
Comment (jacoblee93) [inline docs/api_refs/package.json:31]: Why was this not here before?

File: docs/api_refs/typedoc_plugins/hide_underscore_lc.js (hunk 1 above)
Comment (bracesproul) [reply docs/api_refs/typedoc_plugins/hide_underscore_lc.js:164]: I realized it was unnecessary

File: docs/api_refs/package.json (hunk 3 above)
Comment (bracesproul) [reply docs/api_refs/package.json:31]: We were using npx b/c I didn't want to add a dep (don't remember my reasoning), but I think this is a better practice.

File: docs/api_refs/typedoc_plugins/hide_underscore_lc.js (hunk 2 above)
Comment (bracesproul) [inline docs/api_refs/typedoc_plugins/hide_underscore_lc.js:31]:
```suggestion
<p>This method is deprecated and will be removed in the future.</p>
```

File: docs/api_refs/typedoc_plugins/hide_underscore_lc.js [hunk 4]
```diff
@@ -22,6 +21,17 @@ const SCRIPT_HTML = `<script>
...
 </script>`;
 
+/**
+ * @param {string | undefined} deprecationText 
+ * @returns {string}
+ */
+const DEPRECATION_HTML = (deprecationText) => `<div class="deprecation-warning">
+<h2>⚠️ Deprecated ⚠️</h2>
+${deprecationText ? `<p>${deprecationText}</p>` : ""}
+<p>This method is deprecated and will be removed in the future.</p>
```
Comment (bracesproul) [inline docs/api_refs/typedoc_plugins/hide_underscore_lc.js:31]:
```suggestion
<p>This feature is deprecated and will be removed in the future.</p>
```
